### ISSUE TITLE
Calling "exit 0" from PowerShell makes build fail

### ISSUE THREAD

**Issue by IlyaFinkelshteyn** [appveyor/ci#1090]
Discussion here http://help.appveyor.com/discussions/problems/5363-build-fails-with-command-exited-with-code-without-obvious-errors-and-result-codes

Build fails with message 

> Command exited with code  

without any code.

**Comment by IlyaFinkelshteyn** [comment]
Another issue with probably the same root cause discussed [here](http://help.appveyor.com/discussions/problems/6084-exception-traceback-not-shown-in-log)

It can be easily reposduced by simple adding `Write-Error -Message "123" -ErrorAction Stop` at any stage.

**Comment by zvirja** [comment]
Vote to fix it. We needed to add a workaround in our project because of that, which made our script ugly :(

**Comment by IlyaFinkelshteyn** [comment]
Yes, we have to. But could you look at [this small feature](https://www.appveyor.com/docs/build-worker-api/#forcibly-terminate-current-build-with-success)? I do not know your script, but maybe this can help you make less ugly...

**Comment by zvirja** [comment]
@IlyaFinkelshteyn Thanks for the prompt reply!

Unfortunately, it doesn't help much. I need to abort the `build_script`, while the `test_script` should be still executed. If I got correctly how the suggested termination works, it will abort the whole pipeline.

If you are interested in the particular use-cases for the `exit` keyword to prioritize this issue appropriately, you can check [this](https://github.com/AutoFixture/AutoFixture/commit/a73bad9d94bbfe05578efb95237975c6c6835397) commit.

**Comment by IlyaFinkelshteyn** [comment]
I see, thank you.
